sys-cluster/teleport: add v2.4.0, remove v2.2.*

gclawes opened the pull request:
* Add new release: v2.4.0
* Remove old versions: v2.2.0, v2.2.1, v2.2.4

gclawes commented:
@mgorny done.

mgorny commented on sys-cluster/teleport/teleport-2.4.0.ebuild:
```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2017 Gentoo Foundation
```
Gotta fix your calendar ;-).

mgorny commented on sys-cluster/teleport/teleport-2.4.0.ebuild:
```diff
@@ -0,0 +1,51 @@
+LICENSE="Apache-2.0"
+SLOT="0"
+IUSE=""
+RESTRICT="test strip"
```
Please sort.

mgorny commented on sys-cluster/teleport/teleport-2.4.0.ebuild:
```diff
@@ -0,0 +1,51 @@
+}
+
+src_install() {
+	dodir /var/lib/${PN} /etc/${PN}
```
Why are you doing this? `dodir` is generally to be used only when you need to create a directory into which you're (manually) installing a file, i.e. as replacement for pure `mkdir`. If you need the directory kept, you need `keepdir`. Otherwise it may be stripped as an empty directory.

gclawes commented:
Fixed.  Fixed date, sorted variables, and switched /etc/teleport (config dir) and /var/lib/teleport (data dir) to keepdir.  I mostly saw dodir in various examples I looked at, but keepdir seems a better fit.
